libc/src/sys/mman/linux/CMakeLists.txt

```
@@ -153,6 +153,19 @@ add_entrypoint_object(
     libc.src.errno.errno
 )
 
+add_entrypoint_object(
+  remap_file_pages
+  SRCS
+    remap_file_pages.cpp.cpp
+  HDRS
+    ../remap_file_pages.h.h
```

this is going to cause an error most definitely.
```suggestion
    remap_file_pages.cpp
  HDRS
    ../remap_file_pages.h
```